janusgraph-server/src/test/java/org/janusgraph/graphdb/grpc/schema/SchemaManagerClientTest.java

```diff
@@ -347,4 +348,74 @@ public void testGetEdgeLabels(int numberOfEdges) {
             assertEquals(1, count);
         }
     }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"test", "test2"})
+    public void testGetEdgeLabelByNameEdgeLabelWithEdgeProperty(String propertyName) {
+        final String name = "testEdgeProperty";
+        SchemaManagerClient schemaManagerClient = new SchemaManagerClient(getDefaultContext(), managedChannel);
+
+        //create property
+        EdgeProperty test = EdgeProperty.newBuilder()
+            .setName(propertyName)
+            .setDataType(PropertyDataType.PROPERTY_DATA_TYPE_BOOLEAN)
+            .build();
+        //create edge
+        createEdgeLabel(defaultGraphName, EdgeLabel.newBuilder()
+            .setName(name)
+            .addProperties(test));
+
+        EdgeLabel label = schemaManagerClient.getEdgeLabelByName(name);
+
+        EdgeProperty property = label.getProperties(0);
+        assertEquals(PropertyDataType.PROPERTY_DATA_TYPE_BOOLEAN, property.getDataType());
+        assertEquals(propertyName, property.getName());
+        assertNotEquals(0, property.getId());
+    }
+
+    @ParameterizedTest
+    @EnumSource(value = PropertyDataType.class, mode = EXCLUDE, names = { "PROPERTY_DATA_TYPE_UNSPECIFIED", "UNRECOGNIZED" })
+    public void testGetEdgeLabelByNameEdgeLabelWithEdgeProperty(PropertyDataType propertyDataType) {
+        final String name = "testEdgePropertyType";
+        SchemaManagerClient schemaManagerClient = new SchemaManagerClient(getDefaultContext(), managedChannel);
+
+        //create property
+        EdgeProperty test = EdgeProperty.newBuilder()
+            .setName(name + "-prop")
```

Comment:
(nitpick) Just something for better readability: I would save this name in a variable which can then also be used in the assert in line 396.